### PR TITLE
[dy] Access frontend from server routes

### PR DESCRIPTION
**Pull request by dy46**
# Summary

As title. This will allow users to access the frontend after running `mage start`.

# Tests

tested locally

cc: @wangxiaoyou1993 @johnson-mage

### CODE REVIEW

File: mage_ai/server/server.py [hunk 1]
```diff
@@ -301,6 +306,14 @@ def post(self, kernel_id, action_type):
 def make_app():
     return tornado.web.Application(
         [
+            (r'/', MainHandler),
```
Comment (dy46) [inline mage_ai/server/server.py:309]: the server changes are in this file

File: mage_ai/server/server.py [hunk 2]
```diff
@@ -301,6 +306,14 @@ def post(self, kernel_id, action_type):
...
+            # (r'/pipelines', MainHandler),
```
Comment (wangxiaoyou1993) [inline mage_ai/server/server.py:310]: why do you comment out this?

File: mage_ai/server/server.py [hunk 3]
```diff
@@ -301,6 +306,14 @@ def post(self, kernel_id, action_type):
...
+            (r'/pipelines/(.*)', MainHandler),
+            (
+                r"/_next/static/(.*)",
```
Comment (wangxiaoyou1993) [inline mage_ai/server/server.py:313]: single quote for consistency

File: mage_ai/server/server.py (hunk 2 above)
Comment (dy46) [reply mage_ai/server/server.py:310]: i don't think this route exists yet

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**